### PR TITLE
Relative path fix

### PR DESCRIPTION
**Pull request by Holmsdale**
Fix relative paths specified in the config file not being converted into absolute paths using the correct root in some Linux systems.
Also fixed the template directory not being a sub-directory of the data directory. I may be wrong on this one, but the directory structure and the example conf.ini leaded me to this conclusion.

### CODE REVIEW

File: cert_tools/create_certificate_template.py [hunk 1]
```diff
@@ -137,7 +141,11 @@ def create_certificate_template(config):
     recipient = create_recipient_section(config)
 
     data_dir = config.data_dir
+    if not os.path.isabs(data_dir):
+        data_dir = os.path.join(os.getcwd(), data_dir)
```
Comment (kimdhamilton) [inline cert_tools/create_certificate_template.py]: data_dir = os.path.abspath(data_dir)

**Comment by kimdhamilton** [comment]
Thanks @Holmsdale! Would you mind replacing the lines as shown above?

Per the docs: https://docs.python.org/2/library/os.path.html
```
os.path.abspath(path)
Return a normalized absolutized version of the pathname path. On most platforms, this is equivalent to calling the function normpath() as follows: normpath(join(os.getcwd(), path)).
```

**Comment by Holmsdale** [comment]
Ok! No problem, replaced with abspath

**Comment by kimdhamilton** [comment]
LGTM -- thanks!

**Pull request merged**